api/nakadi-event-bus-api.yaml

```diff
@@ -333,7 +677,8 @@ paths:
         - name: batch_flush_timeout
           in: query
           description: |
-            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached before this
+            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached
+            before this
             time is reached the messages are immediately flushed to the client.
             A value of 0 will flush every batch events immediatly as soon as they come in.
```

Comment:
@valgog 
it's a little bit different (at least in current implementation)
if the batch_limit is not set - it's default value is 1. So the pure flush-on-timeout is not possible.